software/control/gui_hcs.py

```diff
@@ -651,13 +619,16 @@ def makeConnections(self):
 
         self.connectSlidePositionController()
 
-        self.navigationViewer.signal_coordinates_clicked.connect(self.navigationController.move_from_click_mosaic)
+        # TODO(imo): Fix click to move after removal of navigation controller
```

Comment:
All of the "fix click" TODO need to be fixed before merge.  I'm not gonna comment on all of them below.